### PR TITLE
Update Rate Limiter examples and add Deno ones

### PR DESCRIPTION
**Pull request by roj1512**
Related: https://t.me/grammyjs/49604

### CODE REVIEW

**Comment by KnorpelSenf** [comment]
image
Lol

File: site/docs/plugins/ratelimiter.md [hunk 1]
```diff
@@ -46,51 +46,55 @@ There are two ways of using rateLimiter:
...
 
     limit: 3,
 
-    // "MEMORY_STORAGE" is the default mode. Therefore if you want to use Redis, do not pass storageClient at all.
+    // "MEMORY_STORE" is the default value. If you do not want to use Redis, do not pass storageClient at all.
     storageClient: redis,
 
+    // This is called when the limit is exceeded.
     onLimitExceeded: (ctx) => {
       ctx?.reply("Please refrain from sending too many requests!");
```
Comment (KnorpelSenf) [inline site/docs/plugins/ratelimiter.md:99]: May be necessary in further places.
```suggestion
      await ctx?.reply("Please refrain from sending too many requests!");
```

File: site/docs/plugins/ratelimiter.md [hunk 2]
```diff
@@ -99,48 +103,91 @@ bot.use(
     keyGenerator: (ctx) => {
       return ctx.from?.id.toString();
```
Comment (KnorpelSenf) [inline site/docs/plugins/ratelimiter.md:104]: We do not use `return` for promises. This has no advantage, but can be confusing.
```suggestion
    keyGenerator: async (ctx) => {
      await ctx.from?.id.toString();
```

**Review by KnorpelSenf — CHANGES_REQUESTED** [review]

File: site/docs/plugins/ratelimiter.md (hunk 2 above)
Comment (roj1512) [reply site/docs/plugins/ratelimiter.md:104]: The type of `keyGenerator` is `(ctx: C) => string | undefined`.

Comment (KnorpelSenf) [reply site/docs/plugins/ratelimiter.md:104]: I am blind, forget it.

File: site/docs/plugins/ratelimiter.md (hunk 1 above)
Comment (roj1512) [reply site/docs/plugins/ratelimiter.md]: Thanks, I commited modification to them!

**Review by KnorpelSenf — CHANGES_REQUESTED** [review]
Please add JS examples. Right now, the docs mostly have TypeScript/JavaScript/Deno examples. Here, it is Node.js/Deno, so people who aren't on ESM or TS can't copy-paste the examples.

After that, it LGTM.

File: site/docs/plugins/ratelimiter.md [hunk 3]
```diff
@@ -46,101 +46,208 @@ There are two ways of using rateLimiter:
...
-const app = express();
-const bot = new Bot("YOUR BOT TOKEN HERE");
+// Limits message handling to a message per second for each user.
+bot.use(limit());
+```
+
+</CodeGroupItem>
+  <CodeGroupItem title="JavaScript">
+
+```ts
```
Comment (KnorpelSenf) [inline site/docs/plugins/ratelimiter.md:64]:
```suggestion
```js
```

File: site/docs/plugins/ratelimiter.md [hunk 4]
```diff
@@ -46,101 +46,208 @@ There are two ways of using rateLimiter:
...
     },
-  }),
+  })
 );
+```
+
+</CodeGroupItem>
+  <CodeGroupItem title="JavaScript">
+
+```ts
```
Comment (KnorpelSenf) [inline site/docs/plugins/ratelimiter.md:123]:
```suggestion
```js
```

File: site/docs/plugins/ratelimiter.md [hunk 5]
```diff
@@ -46,101 +46,208 @@ There are two ways of using rateLimiter:
...
+      }
+    },
+  }),
+);
+```
+
+</CodeGroupItem>
+  <CodeGroupItem title="JavaScript">
+
+```ts
```
Comment (KnorpelSenf) [inline site/docs/plugins/ratelimiter.md:215]:
```suggestion
```js
```

**Review by KnorpelSenf — CHANGES_REQUESTED** [review]
Please also fix CI.

**Comment by KnorpelSenf** [comment]
> Please also fix CI.

Just saw it's already fixed, perfect!

**Review by KnorpelSenf — APPROVED** [review]

**Comment by KnorpelSenf** [comment]
@roj1512 can you resolve the conflicts?

**Pull request merged**